src/containers/RecievedTrackPanel.tsx

```diff
@@ -0,0 +1,125 @@
+import { JsonComponent } from "../components/JsonComponent";
+import { TrackMetadata } from "../jellyfish.types";
+import { useState } from "react";
+import { TrackEncoding } from "@jellyfish-dev/membrane-webrtc-js";
+import { CopyToClipboardButton } from "../components/CopyButton";
+import { AudioPlayer } from "../components/AudioPlayer";
+import { PiMicrophoneFill } from "react-icons/pi";
+import VideoPlayer from "../components/VideoPlayer";
+type TrackPanelProps = {
+  clientId: string;
+  trackId: string;
+  stream: MediaStream | null;
+  trackMetadata: TrackMetadata | null;
+  changeEncodingRecieved: (trackId: string, encoding: TrackEncoding) => void;
+  vadStatus: string | null;
+  encodingRecieved: TrackEncoding | null;
+};
+
+export const RecievedTrackPanel = ({
+  clientId,
+  trackId,
+  stream,
+  vadStatus,
+  trackMetadata,
+  encodingRecieved,
+  changeEncodingRecieved,
+}: TrackPanelProps) => {
+  const isTalking = (vadStatus: string | null) => {
+    if (vadStatus === "silence") {
+      return false;
+    }
+    return true;
+  };
```

Comment:
It could be simplified and you could move it outside the component 

```suggestion
const isTalking = (vadStatus: string | null) => vadStatus !== "silence";
```